plugins/impress.js

```diff
@@ -22,9 +22,15 @@ class Impress {
     });
   }
 
-  slideCount() {
+  slideCount(enableSubsteps) {
```

Comment:
I think it'd be preferable to have the `options` object passed to the Impress plugin constructor and remove the extra argument.